Makefile.am

```diff
@@ -1,4 +1,4 @@
 
 AUTOMAKE_OPTIONS = dist-bzip2 dist-zip
-SUBDIRS = src doc
+SUBDIRS = src
```

Comment:
see the commit msg, building doc requires makeinfo (from texinfo) which we do not have in dockerize. It doesn't seem reasonable to install texinfo just to build netperf doc, so let's nuke it.